src/Select/index.js

```diff
@@ -121,18 +128,22 @@ const Select = ({
   errorText,
   testId,
 }) => {
-  let items = [];
-  let categories = [];
+  let items = []; // List of all item types to pass to downshift state management
+  let options = []; // All Select.Item options
+  let categories = []; // Categories extracted from Select.Category children
+  let actions = []; // All Select.Action items
   const [userInput, setUserInput] = useState(""); // most recent val the user typed while focused on this input
 
-  const allChildren = React.Children.toArray(children);
+  options = React.Children.toArray(children).filter((item) => !isAction(item));
+  actions = React.Children.toArray(children).filter(isAction);
```

Comment:
Ah, so we can always put the actions at the bottom? Sounds good, that'll help us enforce the design!
How would you feel about making these two `const` instead of `let` to indicate that there's no further reassignment? 🤔

Reply:
Yeah, I can just directly assign these where `items` is initialized... Going to make this change, thanks!